src/vm/eeconfig.cpp

```diff
@@ -789,10 +792,18 @@ HRESULT EEConfig::sync()
         g_fEnableARM = TRUE;
     }
 
-    int forceGCconcurrent = CLRConfig::GetConfigValue(CLRConfig::UNSUPPORTED_gcConcurrent);
-    if ((forceGCconcurrent > 0) || (forceGCconcurrent == -1 && g_IGCconcurrent))
+    bool gcConcurrentWasForced = false;
+#ifdef FEATURE_CORECLR
```

Comment:
We are looking up the `CLRConfig::UNSUPPORTED_gcConcurrent` and `System.GC.Concurrent` both here as well as in `InitializeStartupFlags`. It is redundant.

I think we should either:

- Lookup both in `InitializeStartupFlags` and do nothing here (put this piece of code here under `#ifndef FEATURE_CORECLR` ifdef)

or

- Lookup just `System.GC.Concurrent` in `InitializeStartupFlags` (use the overload that takes default value as constant) and `UNSUPPORTED_gcConcurrent` here - the existing code here should work just fine for it, no need to ifdef it.